modules/checking-cluster-resources-before-initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-cluster-resources-before-initiating-limited-live-migration_{context}"]
+= Checking cluster resources before initiating the limited live migration
+
+The following procedure shows you how to check metrics for EgressIPs resources, EgressFirewalls resources, and multicast enabled namespaces. These resources are migrated to OVN-Kubernetes, however, they might behave differently after migration. Users should check these resources before and after migration to ensure the intended behavior.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Check your cluster for EgressFirewall resources:
+
+.. On the {product-title} web console, navigate to *Observe* -> *Metrics*.
+
+.. In the *Expression* box, enter `sdn_controller_num_egress_firewalls`.
+
+.. Click *Add* -> *Run queries*. Note the intended behavior of this resource, as it might change after migration.
```

Comment:
from this query,  it's only show the nums of egress_firewalls. but cannot show which namespace are using this feature.  I think it's not better for user to check the feature is working well after migration. 

How about using `oc get egressnetworkpolicies.network.openshift.io -A` using admin user.  
@pliurh ^^

Reply:
![Screenshot from 2024-07-03 10-40-11](https://github.com/openshift/openshift-docs/assets/3785307/d2eb9442-b3cf-4832-98cd-0e35c58a5a77)

Reply:
Maybe we can show both approaches here.

Reply:
@zhaozhanqi @pliurh I am confused. Are we using `oc get egressnetworkpolicies.network.openshift.io -A` before or after migration?

Reply:
@stevsmit 
in openshift-sdn the resource is `egressnetworkpolicies` and after migration to ovn, the resource should be migration to egressfirewalls, So 

Before migration check: `oc get egressnetworkpolicies.network.openshift.io -A`
After migration check : `oc get egressfirewalls.k8s.ovn.org -A`

Reply:
@zhaozhanqi I added a new section for checking resources AFTEr migration. See: https://78178--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#checking-cluster-resources-after-initiating-limited-live-migration_migrate-from-openshift-sdn

Should we also tell them to check for multicast/egress IPs after migration?